### ISSUE TITLE
TFX CLI Codelab review

### ISSUE THREAD

**Issue by sayakpaul** [gde-codelabs/gde-codelabs.github.io#5]
In the first step, I think it's a good idea to introduce what the Codelab is all about and who authored it. An example: https://codelabs.developers.google.com/codelabs/cloud-bigquery-python#0. Consider including what would be needed in order to run the Codelab. I think a Codelab is supposed to be an almost realtime engagement. It's different from a tutorial in those scopes. 

I think you can skip explaining the different components since you can always point the readers to the official documentation for that. Prefer brief explanations instead. Most of the Codelabs you will see contain a very compact amount of text. This is another great example: https://codelabs.developers.google.com/codelabs/cloud-tensorflow-mnist#0. 

Maybe you could also rename the Codelab to something like "Introduction to TFX CLI". 

Some popular examples we could refer to improve our overall layout:

* https://github.com/googlecodelabs/automl-vision-edge-in-mlkit

**Comment by deep-diver** [comment]
@sayakpaul 

Wow, thanks for introducing some of the great examples!
Your comments should be listed in the README :) 

Just let me summarize what to keep in mind
- [x] Add authors
- [x] Add environmental setup (requirement)
- [x] Keep the explanation as compact as possible by leaving links of other resources(official doc) to help readers not to lose their focus
- [x] Make the title more noticible

There is one thing to note though. That is our codelabs is not the actual codelabs(hugo with codelab theme), so this doesn't cover all of the md format that the official codelabs provides.

**Comment by sayakpaul** [comment]
> There is one thing to note though. That is our codelabs is not the actual codelabs(hugo with codelab theme), so this doesn't cover all of the md format that the official codelabs provides.

Yeah, that I understand. Since these Codelabs (the ones I mentioned) are widely used by developers all around the globe, I thought they might serve as great reference points for us to prepare and deliver the content in the best possible manner.
